starlette/middleware/database.py

```
@@ -0,0 +1,27 @@
+from functools import partial
```

Ruff (0.11.9)

`functools.partial` imported but unused

Remove unused import: `functools.partial`

(F401)